index.md

```diff
@@ -1,4 +1,127 @@
-# This is the **HOMEPAGE**.
-Refer to [Markdown](http://daringfireball.net/projects/markdown/) for how to write markdown files.
-## Quick Start Notes:
-1. Add images to the *images* folder if the file is referencing an image.
+---
+title: Welcome to the Blazor preview docs website!
+description: Learn about Blazor, an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+---
+<div class="cr cr-top cr-right cr-sticky cr-green">Experimental</div>
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>BLAZOR</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="#">Get Started</a><span>|</span><a class="github-link" href="/docs/get-started.html">View in Github</a>
```

Comment:
The Get Started link here seems redundant with the button. I think it can be removed.